test/realmodel/src/generateCase.js

```diff
@@ -27,15 +15,91 @@ function mkdirsSync(dirname) {
 }
 mkdirsSync(case_path);
 
+let regexmodel = /resnet/;
+let matchFlatmodel = regexmodel.test(`${JSON_DATA.getModelName()}`);
+if (matchFlatmodel) {
+  let filePath = path.join(__dirname, '..', 'model', JSON_DATA.getModelName(), `${JSON_DATA.getModelName()}.json`);
```

Comment:
@shenx3x From L#21 to L#61, many similar blocks, please optimize those codes, thanks